Wox.Plugin/IPublicAPI.cs

```diff
@@ -81,7 +81,16 @@ public interface IPublicAPI
         /// <param name="title">Message title</param>
         /// <param name="subTitle">Message subtitle</param>
         /// <param name="iconPath">Message icon path (relative path to your plugin folder)</param>
-        void ShowMsg(string title, string subTitle = "", string iconPath = "", bool useMainWindowAsOwner = true);
+        void ShowMsg(string title, string subTitle = "", string iconPath = "");
+
+        /// <summary>
+        /// Show message box
+        /// </summary>
+        /// <param name="title">Message title</param>
+        /// <param name="subTitle">Message subtitle</param>
+        /// <param name="iconPath">Message icon path (relative path to your plugin folder)</param>
+        /// <param name="useMainWindowAsOwner">when true will use main windows as the owner</param>
+        void ShowMsg(string title, string subTitle, string iconPath, bool useMainWindowAsOwner = true);
```

Comment:
@theClueless should we not retain default parameters in case other classes call it directly and still rely on defaults

Reply:
Since this overload has defaults you get all the possible defaults:
 void ShowMsg(string title, string subTitle = "", string iconPath = "")

Reply:
What about ShowMsg("title", "subtitle", useMainWindowAsOwner: false)